assets/js/contact.js

```diff
@@ -1,22 +1,38 @@
-function bindCaptchaValidation() {
+function verifyCanSubmitEntry() {
   const debug = Array.prototype.slice.apply(document.getElementsByName('debug'))[0];
   if (debug && +debug.value) {
     return;
   }
 
   const form = document.getElementById('contact-us-form');
   const error = document.getElementById('captcha-error-message');
-
+  const piiError = document.getElementById('pii-warning');
+  const piiErrorText = document.getElementById('pii-warning-message');
+  const descriptionInput = document.getElementById('description');
+  let counter = 0;
```

Comment:
This counter is a bit confusing to me, can we change it to a boolean with a more descriptive name, since it is being used as a bool from what I can tell to confirm the user has already seen/accepted the pii warning.